server.go

```diff
@@ -666,12 +666,18 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	if err != nil {
 		return nil, err
 	}
+
+	interceptorMode, err := s.cfg.ParseHtlcInterceptorMode()
```

Comment:
nit: this function could instead just set a "mode" variable on the config struct and be called in `config.go` rather than here